tests/defaults_list/test_defaults_tree.py

```diff
@@ -1333,6 +1358,37 @@ def test_extension_use_cases(
             ),
             id="nested_missing:override",
         ),
+        param(
+            "empty",
+            ["+group1=with_missing_at_foo"],
+            raises(
+                ConfigCompositionException,
+                match=dedent(
+                    """\
+                    You must specify 'group1/group2@group1.foo', e.g, group1/group2@group1.foo=<OPTION>
+                    Available options"""
+                ),
+            ),
+            id="nested_missing_at_foo",
+        ),
+        param(
+            "empty",
+            ["+group1=with_missing_at_foo", "group1/group2@group1.foo=file1"],
+            DefaultsTreeNode(
+                node=ConfigDefault(path="empty"),
+                children=[
+                    ConfigDefault(path="_self_"),
+                    DefaultsTreeNode(
+                        node=GroupDefault(group="group1", value="with_missing_at_foo"),
+                        children=[
+                            GroupDefault(group="group2", value="file1", package="foo"),
+                            ConfigDefault(path="_self_"),
+                        ],
+                    ),
+                ],
+            ),
+            id="nested_missing_at_foo:override",
+        ),
```

Comment:
Is this not already tested somewhere?

Reply:
No, the combination of `@package` with a missing key `???` is not tested. These tests are failing on the master branch.